packages/helm-charts/oracle/USDCUSD.yaml

```diff
@@ -16,14 +16,19 @@ oracle:
         {exchange: 'KRAKEN', symbol: 'USDCUSD', toInvert: false}
       ],
       [
-        {exchange: 'BITSTAMP', symbol: 'USDCUSD', toInvert: false}
+        {exchange: 'BINANCEUS', symbol: 'USDCUSD', toInvert: false}
       ],
       [
-        { exchange: 'GEMINI', symbol: 'USDCUSD', toInvert: false }
+        {exchange: 'COINBASE', symbol: 'USDTUSDC', toInvert: true},
+        {exchange: 'COINBASE', symbol: 'USDTUSD', toInvert: false}
+      ],
+      [
+        {exchange: 'WhiteBIT', symbol: 'USDCUSDT', toInvert: false},
+        {exchange: 'COINBASE', symbol: 'USDTUSD', toInvert: false}
```

Comment:
was the WhiteBIT:USDTUSD rate bad?

Reply:
Here is the distribution of the deviation from average using WhiteBIT USDTUSD:

![image](https://user-images.githubusercontent.com/53600650/216056774-e47ad192-fe8e-4113-ba84-6d8b679a73a7.png)


And here using Coinbase USDTUSD (sorry, for being too lazy to use same scale and colours...)

!![image](https://user-images.githubusercontent.com/53600650/216057827-2f9d22fb-6113-45aa-98ee-5a4906067a8f.png)

I believe the differences come from trading volumes at WhiteBIT USDTUSD has 2m daily at Coinbase it is 80m

Reply:
ok wow 😅